test/e2e/apps/rc.go

```diff
@@ -139,6 +139,12 @@ var _ = SIGDescribe("ReplicationController", func() {
 
```

Comment:
using raw watches is fragile; those are subject to being dropped and nothing in this test handles re-establishing the watch